apps/src/code-studio/pd/constants.js

```diff
@@ -21,3 +21,5 @@ export const PrivacyDialogMode = utils.makeEnum(
   'TEACHER_APPLICATION',
   'PRINCIPAL_APPROVAL'
 );
+
+export const BuildYourOwnWorkshop = 'Build Your Own Workshop';
```

Comment:
Oh shoot I ended up also adding one too here. I'm happy to remove mine in one of my followup tasks! Do you think it makes more sense to have the constant at the level of `/code-studio/pd/` or /code-studio/pd/workshop_dashboard` ?

Reply:
oh gosh my bad! And oh I don't care either way- I just got excited when I saw that pd shared constants file and threw it in there.

Reply:
Is yours merged yet? I can move this one and update the imports

Reply:
I moved mine to that workshopConstants file!

Reply:
LOL my bad- I hadn't merged staging yet today. Okay I ended up leaving the longer version to match the backed constant, resolved the merge conflict and updated the file you're using it in!

Reply:
Yep mine is merged!

Reply:
Thanks for updating mine too!